docs/gen-docs/kyma_version.md

```diff
@@ -15,13 +15,13 @@ kyma version [flags]
 
 ```
   -c, --client   Client version only (no server required)
-  -h,  --help     Displays help for the command.
+  -h, --help     Displays help for the command.
 ```
 
 ### Options inherited from parent commands
 
 ```
-      --kubeconfig string   Specifies the path to the kubeconfig file. (default KUBECONFIG environment variable or "/$HOME/.kube/config" if KUBECONFIG is not set)
+      --kubeconfig string   Specifies the path to the kubeconfig file. Use the default KUBECONFIG environment variable or /$HOME/.kube/config if KUBECONFIG is not set.
```

Comment:
The change to this sentence is a bit misleading, I understand it as: I (the user) have to use the KUBECONFIG or the /HOME... Also "default KUBECONFIG" is confusing, there is no such thing.
It should somehow make clear that the CLI will automatically picks either the KUBECONFIG or the /$HOME/.kube/config if not KUBECONFIG was found when this flag is not explicitly set (so it is the default value).